app/scripts/views/sign_up.js

```diff
@@ -197,7 +198,24 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     },
 
     suggestEmail: function () {
-      mailcheck(this.$el.find('.email'), this.metrics, this.translator, this.window.location.search);
+      var MAILCHECK_ON = 'true';
+      var abChoose;
+      var abData = {
+        isMetricsEnabled: !! this.metrics._able,
```

Comment:
Should we add an `isCollectionEnabled` or some other function to metrics so we don't have to peek at `this.metrics._able`?